fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/BindPredicate.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+
+/**
+ * Currently modeled as a unary predicate rather than a literal until we model
+ * equality and inequality operators
```

Comment:
Is this actually needed?  I saw the quality operators and inequality operators in the operator enum and used in the adapter.

Reply:
Predicate hierarchy will be removed in #2321.